lib/sorbet-rails/model_plugins/active_record_querying.rb

```diff
@@ -33,7 +33,6 @@ def generate(root)
         method_name.to_s,
         parameters: [
           Parameter.new("*args", type: "T.untyped"),
-          Parameter.new("&block", type: "T.nilable(T.proc.void)"),
```

Comment:
I think all querying methods are defined here:

https://github.com/rails/rails/blob/9a7f15c905c9ca7a26b56b6f2a045cf56805a99e/activerecord/lib/active_record/relation/query_methods.rb

I did a grep of block and find that other than `select`, `extending` is the only other method accepting a block
https://github.com/rails/rails/blob/9a7f15c905c9ca7a26b56b6f2a045cf56805a99e/activerecord/lib/active_record/relation/query_methods.rb#L918

Can you separate it from the rest?

Reply:
Thanks for catching that!

Reply:
Looks good now! thanks for fixing!